easybuild/tools/github.py

```diff
@@ -1611,10 +1611,10 @@ def check_github():
     # check whether we're online; if not, half of the checks are going to fail...
     try:
         print_msg("Making sure we're online...", log=_log, prefix=False, newline=False)
-        urlopen(GITHUB_URL, timeout=5)
+        urlopen(GITHUB_URL, timeout=30)
```

Comment:
Should we be checking `GITHUB_API_URL` here instead?

Reply:
That shouldn't hurt, but it may not matter.

Checking `https://github.com` has worked fine until now, so why is it broken now?

@Flamefire Can you change the PR to use `GITHUB_API_URL` instead, to see if it helps?

Reply:
I had the same idea, but we also use GITHUB_URL. Another test incoming

Reply:
See https://github.com/easybuilders/easybuild-framework/pull/3192/commits/c495404a24425a3368e11b28a07454932debfe2f